kiwi_lint/empty.py

```diff
@@ -73,3 +75,23 @@ def close(self):
         for fname in diff:
             fname = fname.replace(dir_prefix, '')
             self.add_message('module-in-directory-without-init', args=(fname,))
+
+
+class EmptyClassChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker,)
+
+    name = 'empty-class-checker'
+
+    msgs = {'E4483': ("Remove empty class from git!",
+                      'remove-empty-class',
+                      "Kiwi TCMS doesn't need to carry around classes which are empty. "
+                      "They must be removed from the source code!")}
+
+    @utils.check_messages('remove-empty-class')
+    def visit_classdef(self, node):
+        if not node.body:
+            self.add_message('remove-empty-class', node=node)
+
+        for child in node.body:
```

Comment:
FTR this is slightly incorrect b/c it will produce a pylint error also for classes which contain `pass` somewhere in the class body even if there are other attributes. However that is also probably an error (left over) so we should be fine.